extensions/wikia/ArticleComments/modules/ArticleCommentsController.class.php

```diff
@@ -1,50 +1,14 @@
 <?php
 class ArticleCommentsController extends WikiaController {
+	use Wikia\Logger\Loggable;
+
 	private $dataLoaded = false;
 	private static $content = null;
 
 	public function executeIndex() {
-		wfProfileIn(__METHOD__);
-
-		if (class_exists('ArticleCommentInit') && ArticleCommentInit::ArticleCommentCheck()) {
+		if ( class_exists( 'ArticleCommentInit' ) && ArticleCommentInit::ArticleCommentCheck() ) {
 			$isMobile = $this->app->checkSkin( 'wikiamobile' );
 
-			// for non-JS version !!! (used also for Monobook and WikiaMobile)
```

Comment:
WikiaMobile now seems to have an AC JS package and the Oasis JS also gets added to Monobook. Should I re-add it so that it works if JS is disabled for some reason?